test/conformance/testdata/conformance.yaml

```diff
@@ -1191,8 +1191,7 @@
   release: v1.20
   file: test/e2e/instrumentation/core_events.go
 - testname: DNS, cluster
-  codename: '[sig-network] DNS should provide /etc/hosts entries for the cluster [LinuxOnly]
-    [Conformance]'
+  codename: '[sig-network] DNS should provide /etc/hosts entries for the cluster [Conformance]'
   description: When a Pod is created, the pod MUST be able to resolve cluster dns
     entries such as kubernetes.default via /etc/hosts.
   release: v1.14
```

Comment:
I wonder if release needs to be changed in this case. To some extend it is basically a new test for Windows starting 1.24, while old test keeps working starting 1.14

Reply:
following up should be ok